src/test/java/com/example/amazoncom/amazontest.java

```diff
@@ -7,51 +7,38 @@
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
-import org.openqa.selenium.support.PageFactory;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
-
 import java.time.Duration;
 
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public class amazontest {
     public WebDriver driver;
+    AmazonMainPage mainPage;
 
+    final static String WEB="https://www.amazon.es/";
     @BeforeAll
     public void setupBrowser() {
-        System.out.println("Test starts in amazon.es");
         WebDriverManager.chromedriver().setup();
         ChromeOptions chromeOptions = new ChromeOptions();
         chromeOptions.addArguments("incognito");
+        //chromeOptions.addArguments("headless");
         driver = new ChromeDriver(chromeOptions);
-        driver.manage().window().maximize();
-        driver.get("https://www.amazon.es/");
-        this.driver = driver;
-        WebElement waiting = (new WebDriverWait(driver, Duration.ofSeconds(10)).until(ExpectedConditions.presenceOfElementLocated((By.xpath("//input[contains(@id,'twotabsearchtextbox')]")))));
+        driver.get(WEB);
+        mainPage = new AmazonMainPage(driver);
+ //       WebElement waiting = (new WebDriverWait(driver, Duration.ofSeconds(30)).until(ExpectedConditions.presenceOfElementLocated((By.xpath("//input[contains(@id,'twotabsearchtextbox')]")))));
+    }
+    @AfterEach
+    public void teardown() {
+      //  driver.quit();
     }
-
     @Test
-    public void myTest() {
-        //   WebDriver driver=this.driver;
+    public void search() {
```

Comment:
Use the test idea in the method name: searchProductTest(), and use "test" at the end of the test method.